docs/Configuring.md

```diff
@@ -148,6 +148,32 @@ On every poll, Cromwell will take at limited number of new submissions, provided
 system.max-workflow-launch-count = 50
 ```
 
+***Abort configuration***
+
+Cromwell will scan for abort requests using default configuration values equivalent to those below. In most circumstances
+there shouldn't be a need to override these defaults.
```

Comment:
Would it be better to link to the appropriate section in `reference.conf` - just in case this changes over time?